apps/web/src/pages/dao/[token]/[tokenId].tsx

```diff
@@ -161,6 +169,8 @@ export const getServerSideProps: GetServerSideProps = async ({
       `public, s-maxage=${maxAge}, stale-while-revalidate=${swr}`
     )
 
+    getFarcasterProfile(3)
```

Comment:
do we need this call to `getFarcasterProfile` here

Reply:
Whoops!